src/components/App/App.js

```diff
@@ -4,6 +4,7 @@ import Grid from '../Grid/Grid'
 import Navigation from '../Navigation/Navigation'
 import VotingPage from '../VotingPage/VotingPage'
 import { useState } from 'react'
+import AppHeader from '../Header/Header'
```

Comment:
```suggestion
import Header from '../Header/Header'
```

Isn't it exported as Header?